### ISSUE TITLE
[Bug]: locales-utf8 check on macOS results in error during requirements check

### ISSUE THREAD

**Issue by rolfhermancoen** [pimcore/pimcore#15505]
### Pimcore version

11.0.3

### Steps to reproduce

- Run ./vendor/bin/pimcore-install on a macOS Ventura 13.4.1 system with either the pimcore-skeleton or pimcore-demo.

### Expected / actual behavior

**Expected**:
The installer works for the locales that are on macOS Ventura 13.4.1.

**Actual Behaviour:**
The check that is currently run during installation checks for ['en.utf8', 'en_US.utf8', 'en_GB.utf8'], which are not present on the system, the notation of those locales are different on macOS -> ['UTF-8', 'en_US.UTF-8', 'en_GB.UTF-8']. This will therefore always result in a failed check and failed installation

These are available for macOS for en:
```
ls -la | grep en
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 en_AU
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 en_AU.ISO8859-1
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 en_AU.ISO8859-15
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 en_AU.US-ASCII
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 en_AU.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 en_CA
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 en_CA.ISO8859-1
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 en_CA.ISO8859-15
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 en_CA.US-ASCII
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 en_CA.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 en_GB
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 en_GB.ISO8859-1
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 en_GB.ISO8859-15
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 en_GB.US-ASCII
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 en_GB.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 en_IE
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 en_IE.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 en_NZ
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 en_NZ.ISO8859-1
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 en_NZ.ISO8859-15
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 en_NZ.US-ASCII
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 en_NZ.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 en_US
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 en_US.ISO8859-1
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 en_US.ISO8859-15
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 en_US.US-ASCII
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 en_US.UTF-8
```

And for UTF-8:

```
ls -la | grep UTF-8
drwxr-xr-x    3 root  wheel    96 Jun 15 12:08 UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 af_ZA.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 am_ET.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 be_BY.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 bg_BG.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 ca_ES.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 cs_CZ.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 da_DK.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 de_AT.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 de_CH.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 de_DE.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 el_GR.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 en_AU.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 en_CA.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 en_GB.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 en_IE.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 en_NZ.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 en_US.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 es_ES.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 et_EE.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 eu_ES.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 fi_FI.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 fr_BE.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 fr_CA.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 fr_CH.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 fr_FR.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 he_IL.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 hr_HR.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 hu_HU.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 hy_AM.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 is_IS.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 it_CH.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 it_IT.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 ja_JP.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 kk_KZ.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 ko_KR.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 lt_LT.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 nl_BE.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 nl_NL.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 no_NO.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 pl_PL.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 pt_BR.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 pt_PT.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 ro_RO.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 ru_RU.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 sk_SK.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 sl_SI.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 sr_YU.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 sv_SE.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 tr_TR.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 uk_UA.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 zh_CN.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 zh_HK.UTF-8
drwxr-xr-x    8 root  wheel   256 Jun 15 12:08 zh_TW.UTF-8
```

**Comment by kingjia90** [comment]
@rolfhermancoen Thank you for the report!
Can you please try if these changes on https://github.com/pimcore/pimcore/pull/15612 will fix the issue? Not having a macOS to test it but should work fine

**Comment by ghost** [comment]
I added those 2 small changes in my vendor folder and the requirement check no longer fails. Thanks.

**Comment by dvesh3** [comment]
Fixed by #15612
